src/commands.h

```diff
@@ -39,4 +39,12 @@
 
 #define RS_DEBUG RS_CMD_PREFIX ".DEBUG"
 
+#define RS_SPELL_CHECK RS_CMD_PREFIX ".SPELLCHECK"
+
+#define RS_DICT_ADD RS_CMD_PREFIX ".DICTADD"
```

Comment:
This should be prefixed with something like SC_ or SPELLDICT, etc. in case we have another dictionary in the future to interface with.

Reply:
I am not sure, this dictionary can serves more functionalities other then SC so why limit it?